### ISSUE TITLE
How to use mip start?

### ISSUE THREAD

**Issue by tosttost** [ERGO-Code/HiGHS#719]
There is https://github.com/ERGO-Code/HiGHS/pull/695/commits/59dc0552be9bb59c44db8bc268cab30b114e316b but I'm not sure how to use this (using C++).

I tried
```
Highs solver;
solver.passModel(/* some mip */);
HighsSolution solution;
solution.col_value = { 1, 2, 3 };
solver.setSolution(solution);
solver.run();
```
but the solution is not used because
```
if (solution.value_valid) {
```
in the ctor of HighsMipSolver fails (due to missing duals?).

Is mip start meant to be usable in v1.2? If yes how do I use it?

**Comment by jajhall** [comment]
Indeed, https://github.com/ERGO-Code/HiGHS/commit/59dc0552be9bb59c44db8bc268cab30b114e316b suggests that this should work. It sounds as if @lgottwald and @jajhall need to look into this.

**Comment by lgottwald** [comment]
MIP solver expects the Highs class to make sure row values are computed and hence value_valid is set.

**Comment by jajhall** [comment]
Thanks, I realised that I'd omitted something. Currently fixing #717 and will then address this

**Comment by jajhall** [comment]
This is now fixed, and will be available in v1.2.1
